### DISCUSSION TITLE
Autolaunch from runway or water

### DISCUSSION THREAD

**Discussion by swissembedded** [iNavFlight/inav#7505] [Ideas]
Autolaunch is triggered by acceleration forces of the throw. Is there a way to trigger launch with a switch, thinking of launching fixed wing with wheels on a runway or water. Is it also possible to use rudder stick without terminating autolaunch sequence?

**Comment by wernight** [comment]
It's an interesting idea I wondered myself, however practically it seems hard to do right with few advantages. A runway, even in poor conditions requires the pilot to follow it and speed up until take off speed. All these are hard to automate. Also it's relatively easy to take off on a runway. Easier than throwing.

**Comment by breadoven** [comment]
You could try using the manual autolaunch option. It just controls the climb out, throttle is manually controlled. So rather than using a switch you just raise the throttle ... and see what happens. In the HITL sim it lifts off and climbs the same as a normal autolaunch, Autolaunch is only terminated using the Roll/Pitch stick.
